File: dashlane-project/Dashlane/src/main/java/com/dashlane/logger/utils/LogsSender.java
package com.dashlane.logger.utils;

import com.dashlane.dagger.singleton.SingletonProvider;
import com.dashlane.logger.Log;

public class LogsSender {

    private LogsSender() {
        
    }

    public static void flushLogs() {
        Log.d("TRACKING", "Send logs");
        SingletonProvider.getComponent().getUserActivityFlush().invoke();
        SingletonProvider.getComponent().getLogFlush().invoke();
    }
}

File: dashlane-project/Dashlane/src/main/java/com/dashlane/logger/utils/LogsSender.kt
package com.dashlane.logger.utils

import com.dashlane.hermes.LogFlush
import com.dashlane.logger.Log.d
import javax.inject.Inject

class LogsSender @Inject constructor(
    private val logFlush: LogFlush
) {
    fun flushLogs() {
        d("TRACKING", "Send logs")
        logFlush.invoke()
    }
}